app/src/main/res/layout/fragment_drawer.xml

```diff
@@ -0,0 +1,38 @@
+<com.google.android.material.navigation.NavigationView xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:id="@+id/nav_view"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:layout_gravity="start"
+  android:fitsSystemWindows="true"
+  app:headerLayout="@layout/nav_header_navigation_drawer"
+  app:itemBackground="@android:color/transparent"
+  app:itemIconTint="@color/drawer_item"
+  app:itemTextColor="@color/drawer_item"
+  app:menu="@menu/navigation_drawer_menu">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_gravity="bottom"
+    android:orientation="horizontal"
+    android:padding="16dp">
+
+    <ImageView
+      android:layout_width="@dimen/image_dim_24"
+      android:layout_height="@dimen/image_dim_24"
```

Comment:
If it is fix 24dp then suggest declaring it directly here and do not declare this in dimens file.

Reply:
Done